src/tasks/scenarios/project.ts

```diff
@@ -1,32 +1,80 @@
 import config = require('../../../config.js')
-import {cat, exec, mkdir, rm, touch, head} from 'shelljs'
-import { ProjectJob } from 'tasks/job'
+import { cat, exec, mkdir, rm, touch, head } from 'shelljs'
+import { ProjectJob } from '../jobs/project'
 import { ProjectResult } from 'types/result'
 import * as path from 'path'
-import * as fs from 'fs'
 import { Scenario } from 'tasks/scenario'
 import { download } from 'utils/request'
+const AdmZip = require('adm-zip')
 
 export default class ProjectScenario extends Scenario {
   async setup(currentJobDir: string, job: ProjectJob) {
-    // TODO
-    const problemDir = path.join(currentJobDir, job.problem)
-    await download(job.problem, problemDir)
 
-    const solutionDir = path.join(currentJobDir, job.source)
-    await download(job.problem, solutionDir)
+    // for testing only, change it to download
+    var problemZip = new AdmZip('/judge-worker-prabal/current/problem.zip')
+    var solutionZip = new AdmZip('/judge-worker-prabal/current/solution.zip')
+
+    const problemDir = path.join(currentJobDir, 'problem')
+    mkdir('-p', problemDir)
+    problemZip.extractAllTo(problemDir, true);
+
+    const solutionDir = path.join(currentJobDir, 'solution')
+    mkdir('-p', solutionDir)
+    solutionZip.extractAllTo(solutionDir, true);
+  }
+
+  run(currentJobDir: string, job: ProjectJob) {
+
+    // LANG_CONFIG is undefined rn, hence card coding the value of cpus and memory
+    const LANG_CONFIG = config.LANGS[job.lang]
```

Comment:
We will be creating a new config here named PROJECTS. In future, we might need more configurable information for these projects.